doc/releases/release-notes-2.5.rst

```diff
@@ -759,6 +759,14 @@ Libraries / Subsystems
 
 * Power management
 
+  * Use a consistent naming convention using **pm_** namespace.
+  * Overhaul power states. New states :c:enum:`pm_state` are more
+    meaningful and ACPI alike.
+  * Move residency information and supported power states to devicetree.
+    * Remove related Kconfig options
```

Comment:
Looks like an indentation problem here.  Or if this is supposed to be a single subordinate bullet point, it's probably best to integrate it into the sentence ("... to devicetree and remove related Kconfig options").